.github/workflows/test.yml

```diff
@@ -73,73 +86,110 @@ jobs:
             cbbr:
               - products/cbbr/**
               - python/**
-            cpdb:
-              - products/cpdb/**
-              - python/**
             checkbook:
               - products/checkbook/**
               - python/**
-            facilities:
-              - products/facilities/**
-              - python/**
-            knownprojects:
-              - products/knownprojects/**
-              - python/**
             zap:
               - products/zap-opendata/**
               - python/**
-      - name: Confirm changes
+      - name: Confirm changes and set matrix
+        id: set-matrix
         run: |
           echo "path filters with changed files: ${{ steps.filter.outputs.changes }}"
-  dcpy:
-    needs: check_changes
-    if: github.event_name != 'pull_request' || contains(needs.check_changes.outputs.path_filters, 'dcpy')
-    uses: ./.github/workflows/dcpy_test.yml
-    secrets: inherit
-  build:
+          filter=$(echo '${{ steps.filter.outputs.changes }}' | sed 's/[][]//g' | sed -r 's/(\w+)(,|$)/"\1"\2/g')
```

Comment:
gets just comma delimited, quoted fields from the path-filter step. Looking now actually it looks like there's a bug. Will fix now